Makefile

```diff
@@ -58,6 +65,20 @@ wheel_manylinux32 wheel_manylinux64: dist/lxml-$(LXMLVERSION).tar.gz
 		$(if $(patsubst %32,,$@),$(MANYLINUX_IMAGE_X86_64),$(MANYLINUX_IMAGE_686)) \
 		bash /io/tools/manylinux/build-wheels.sh /io/$<
 
+wheel_manylinuxaarch64: dist/lxml-$(LXMLVERSION).tar.gz qemu-user-static
```

Comment:
Could you try if this can be folded into the above target? It would be nice to avoid duplicating more than half of the options here, just to add 3 more lines.